text/0000-pack-direct-podman-call.md

```diff
@@ -0,0 +1,86 @@
+# Meta
+[meta]: #meta
+- Name: Pack direct podman call
+- Start Date: 2023-06-16
+- Author(s): dvaumoron
+- Status: Draft
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: "N/A"
+
+# Summary
+[summary]: #summary
+
+A flag `--container-engine` will be added to the `pack` CLI, the allowed flag value will be `docker` (the current functioning) and `podman` to call podman directly (without socket call)
+
+# Definitions
+[definitions]: #definitions
+
+N/A
```

Comment:
The underlying container runtime is `containerd` or `runc` or similar.  I think here we're discussing an alterative to the Docker-API-via-socket with the Podman API.  Is "Orchestration Client API" a good collective term for these?  Is there a more commonly used term?

```suggestion
Orchestration Client API: an API such as Docker's `DockerClient` or Podman's `Runtime` that 
```

Reply:
I am not sure that  "Orchestration Client API" would be a good choice, "Container Orchestration" refer to tools like Kubernetes or Docker Swarm

Reply:
Maybe "Container Manager API" to distinguish from "Container Runtime API" which are lower level API